docs/mkdocs/docs/development/quick-start.md

```
@@ -0,0 +1,205 @@
+You also can run `pytest` without activating environment via Poetry, for example:
+
+```bash
+$ poetry run pytest
+```
+
+### Behave
+
+To run functional tests using `behave`, execute the following command in the terminal:
+
+```bash
+$ behave
```

Markdownlint (0.37.0)

Dollar signs used before commands without showing output

(MD014, commands-show-output)